### PR TITLE
Improve Control of SimulationDuration

### PR DESCRIPTION
**Pull request by kdnilsen**
1. Improve README.md descriptions of InitializationDelay and SimulationDuration.
2. Modify stress workload command line to depend on fewer OS threads.
3. Change implementation of CustomerThread.java and ServerThread.java so that the cyclic execution halts before any sleep request that would not wake up until after the time at which the simulation is intended to terminate.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Extremem/README.md [hunk 1]
```diff
@@ -28,8 +28,11 @@ Command-line arguments configure Extremem to represent different combinations of
 
 It is important to complete all initialization of all global data structures before beginning to execute the experimental workload threads.  If
 initialization is still running when the workload begins to execute, the
-simulation will abort.  Increase this value if the default delays are not
-sufficient.
+simulation will abort.  This value is added to the simulation startup delay
```
Comment (AzeemJiva) [inline Extremem/README.md:31]: Line breaks are unneeded

**Comment by AzeemJiva** [comment]
I don't see evidence of the second item on your list.  Missing commit maybe?

File: Extremem/README.md (hunk 1 above)
Comment (kdnilsen) [reply Extremem/README.md:31]: I realize the file formatting is a bit inconsistent with regards to line breaks.  But I believe the markup renderer doesn't care.

**Comment by kdnilsen** [comment]
> I don't see evidence of the second item on your list. Missing commit maybe?

See lines 170 and 171 of README.md.  This "stress" workload is just a recommended command line.  Volker Simonis suggested that typical hosts are not able to run 20,000 OS threads, so I decreased the number of threads while increasing the amount of work performed by each thread proportionately.

**Review by AzeemJiva — APPROVED** [review]

**Pull request merged**